charts/datadog/templates/rbac.yaml

```diff
@@ -129,6 +129,13 @@ rules:
   resources: ["secrets"]
   verbs: ["get"]
 {{- end }}
+- apiGroups:  # EKS kube_scheduler and kube_controller_manager control plane metrics
+  - "metrics.eks.amazonaws.com"
+  resources:
+  - kcm/metrics
+  - ksh/metrics
+  verbs:
+  - get
```

Comment:
I wasn't sure but this clusterrole is also use for the cluster-check-runner service account https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/agent-clusterchecks-rbac.yaml